apps/test-suite/tests/ImagePicker.js

```diff
@@ -1,13 +1,19 @@
 import Constants from 'expo-constants';
 import * as ImagePicker from 'expo-image-picker';
 import * as Permissions from 'expo-permissions';
-import { Platform } from 'react-native';
+import { Alert, Platform } from 'react-native';
 
 import * as TestUtils from '../TestUtils';
 import { isDeviceFarm } from '../utils/Environment';
 
 export const name = 'ImagePicker';
 
+async function alertWithInstructions(instructions) {
```

Comment:
Might be worth moving to TestUtils or somewhere else so we can reuse it in other tests. I think we should show alerts also on some other interactive tests.